Makefile

```diff
@@ -36,3 +36,7 @@ clean:
 run:
 	@echo "Starting flask server, please wait..."
 	.venv/bin/python ./flask_server.py
+
+# Run entire setup on docker
+docker-run:
```

Comment:
Does an invocation of `docker compose up` really belong in `Makefile`? The command itself is standard, and it doesn't help to abstract Docker Compose, because to do anything meaningful beyond starting, developers will have to use `docker compose ...` anyway. Then, what about `docker compose down`? IMO it's better to note that there's a configuration that can be used with `docker compose` in `README.md` (or other documentation for developers).

Reply:
I don't hold a strong opinion other than that it should be consistent :)